### DISCUSSION TITLE
Configuration error: time zone is not set.

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1225] [Q&A] [answered]
I have just installed TSML on a new test site.
I have successfully imported my meetings from another site, after using the export function. Which is excellent, that that works!
I have not connected the meeting pages to any menus yet.
But when I used the option 'View posts' from the black top bar, this error came up:
Configuration error: time zone is not set.

I checked my Wordpress settings and it says my timezone is UTC + 0 so it seems there is a timezone configured.
Then I checked my Mac Settings and it is set to configure automatically and sees I am in the general Toronto area.

So what do I do with this error that prevents me from seeing the meetings in the front end? I have looked all over the TSML settings but do not find anywhere to set a timezone. Have not had to do that before either. So I am hoping someone knows what it going on....

Thanks,
Nicky

**Comment by joshreisner** [comment, marked as answer]
hi nicky, it sounds like you have TSML UI enabled. one option is to disable it, another is to set the time zone to be `Toronto` in WordPress settings, rather than `UTC +0`

TSML UI needs to know where to start the list at

**Comment by OSAwebsite** [comment]
Thanks, I'll look into it.

**Comment by OSAwebsite** [comment]
Yep, that fixed the issue. Thanks, Josh.
